### ISSUE TITLE
[test] Add Python 3.6 to CI matrix

### ISSUE THREAD

**Issue by sbellem** [bigchaindb/bigchaindb#1331]

**Comment by morrme** [comment]
Does this just involve editing .travis.yml ?

**Comment by sbellem** [comment]
> Does this just involve editing .travis.yml ?

The `tox.ini` as well I think, (see https://github.com/bigchaindb/bigchaindb/blob/master/tox.ini#L3).

**Comment by morrme** [comment]
OK, so add `36` to the `enlist `  on that line, and what about the `base `and `deps` ? Do they remain the same? 

And for travis.yml, I'd be adding 3.6 to  https://github.com/bigchaindb/bigchaindb/blob/master/.travis.yml#L7    

Do I also add anything to [the include section?](https://github.com/bigchaindb/bigchaindb/blob/master/.travis.yml#L22)

**Comment by sbellem** [comment]
> and what about the baseand deps ? Do they remain the same?

yes, for now, we can update them in a separate PR

**Comment by sbellem** [comment]
> And for travis.yml, I'd be adding 3.6 to https://github.com/bigchaindb/bigchaindb/blob/master/.travis.yml#L7
> 
> Do I also add anything to the include section?

Yes, we need to add two new blocks, one for rethinkdb and one for mongodb.

**Comment by morrme** [comment]
@sbellem Thanks! I am on it!
